int16 motor1
int16 motor2
int16 motor3
int16 motor4
int16 motor5
int16 motor6
---

